internal/uow/exec_test.go

```
@@ -0,0 +1,588 @@
+				require.True(t, ok)
+				require.NotEmpty(t, tx)
+
+				assert.Equal(t, &tt.wantTx, tx)
+			}
+
+			assert.True(t, tt.driver.(*mockStorage).getExecCalled())
+		})
+	}
+}
+
+func TestExecWithRollback(t *testing.T) {
```

golangci-lint (2.5.0)

[error] Function 'TestExecWithRollback' is too long (63 > 60)

(funlen)